px-breadcrumbs.es6.js

```diff
@@ -72,224 +64,137 @@
        */
       clickOnlyMode: {
         type: Boolean,
-        value: false
+        value: false,
+        observer: '_getBreadcrumbsObj'
       },
-      filterMode: {
+      searchMode: {
         type: Boolean,
         value: false
       },
-      _filterString: {
-        type: String,
-        value: ''
-      },
       _hideNoResultLi: {
         type: Boolean,
         value: true
+      },
+      _breadcrumbsObj: {
+        type: Object
       }
     },
-    behaviors: [Polymer.IronResizableBehavior],
```

Comment:
Not sure what you mean. Don't you have to include the behavior in order to use it?